ajax/libs/workbox/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "workbox",
+  "version": "3.0.0",
+  "filename": "workbox-sw.js",
+  "license": "Apache-2.0",
+  "author": "Google's Web DevRel Team",
+  "description": "This module makes it easy to get started with the Workbox service worker libraries.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/GoogleChrome/workbox"
+  },
+  "keywords": [
+    "workbox",
+    "workboxjs",
+    "service worker",
+    "sw",
+    "service-worker",
+    "progressive-web-app",
+    "offline-first",
+    "service-workers",
+    "sw-precache",
+    "sw-toolbox"
+  ],
+  "npmName": "workbox-sw",
+  "npmFileMap": [
+    {
+      "basePath": "build",
+      "files": [
+        "**/!(*workbox-sw.dev*)"
```

Comment:
Would you please let me know that we can use `mjs` in web front-end? Like `ajax/libs/workbox/0.0.1/modules/workbox-sw.prod.v0.0.1.mjs`

Also, Would you please add the old versions files without version with file name? This should be created another commit.

Should we need to add the files under `test` in v1.3.0~v2.1.3? Is there any reference?

Many thanks